EIPS/eip-721.md

```diff
@@ -0,0 +1,312 @@
+
+## Motivation
+
+A standard interface allows wallet/broker/auctioneer applications to work with any deed on Ethereum. This contract considers simple deed contracts as well as contract that track large number of deeds. Additional applications are discussed below.
+
+This standard is inspired by [the ERC-20 token standard](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-20-token-standard.md) and builds on two years of experience since EIP-20 was created. EIP-20 is insufficient for tracking deed ownership because each deed is distinct (non-fungible) whereas each token is identical (fungible).
+
+Differences between this standard and EIP-20 are examined below.
+
+## Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt).
+
+The **baseline specification** is REQUIRED for all ERC-721 implementations (subject to "caveats", below).
+
+```solidity
+pragma solidity ^0.4.19;
+
+/// @title Interface for contracts conforming to ERC-721: Deed Standard
+/// @author William Entriken (https://phor.net), et. al.
+/// @dev Specification at https://github.com/ethereum/eips/XXXFinalUrlXXX
+interface ERC721 {
+
+    // COMPLIANCE WITH ERC-165 (DRAFT) /////////////////////////////////////////
+
+    /// @dev ERC-165 (draft) interface signature for itself
+    // bytes4 internal constant INTERFACE_SIGNATURE_ERC165 = // 0x01ffc9a7
+    //     bytes4(keccak256('supportsInterface(bytes4)'));
+
+    /// @dev ERC-165 (draft) interface signature for ERC721
+    // bytes4 internal constant INTERFACE_SIGNATURE_ERC721 = // 0xe55729f5
+    //     bytes4(keccak256('ownerOf(uint256)')) ^
+    //     bytes4(keccak256('countOfDeeds()')) ^
+    //     bytes4(keccak256('countOfDeedsByOwner(address)')) ^
+    //     bytes4(keccak256('deedOfOwnerByIndex(uint256)')) ^
+    //     bytes4(keccak256('approve(address,uint256)')) ^
+    //     bytes4(keccak256('takeOwnership(uint256)'));
+
+    /// @notice Query a contract to see if it supports a certain interface
+    /// @dev Returns `true` the interface is supported and `false` otherwise,
+    ///  returns `true` for INTERFACE_SIGNATURE_ERC165 and
+    ///  INTERFACE_SIGNATURE_ERC721, see ERC-165 for other interface signatures.
+    function supportsInterface(bytes4 _interfaceID) external pure returns (bool);
+
+    // PUBLIC QUERY FUNCTIONS //////////////////////////////////////////////////
+
+    /// @notice Find the owner of a deed
+    /// @param _deedId The identifier for a deed we are inspecting
+    /// @dev Deeds assigned to zero address are considered destroyed, and
```

Comment:
Does the implementation need to keep the distinction of destroyed v.s. not-yet-created deeds?

Reply:
It does NOT need to track invalid (destroyed) contracts. Thanks for showing this. Please see updates in 6754461.